WooCommerce/Classes/ViewRelated/Coupons/EnhancedCouponListView.swift

```diff
@@ -1,9 +1,9 @@
 import SwiftUI
 
-struct CouponListView: UIViewControllerRepresentable {
+struct EnhancedCouponListView: UIViewControllerRepresentable {
     let siteID: Int64
 
-    typealias UIViewControllerType = CouponListViewController
+    typealias UIViewControllerType = EnhancedCouponListViewController
```

Comment:
I think this is unused: 
```suggestion

```

Reply:
Good point, done in 0ba1ecdc8ee0422981f2a1ad26530f397f366c7d